frontend/test/metabase-db/mongo/query.cy.spec.js

```diff
@@ -6,11 +6,13 @@ import {
   addMongoDatabase,
 } from "__support__/cypress";
 
+const MONGO_DB_NAME = "QA Mongo4";
```

Comment:
Is this an overkill to contain DB name in a `const`, although `addFooDatabase()` function already has the default name value?

My reasoning for this extra step was that it's better to use variable instead of manually searching for database name in every single test.
`cy.findByText("Some Database Name").click()` vs `cy.findByText(DB_NAME).click()`.

Also, this approach allows the person who is writing a test to use their own name, if they wish so.

Reply:
Since the database name is repeated a few times, it's always a good idea anyway to turn it into a constant/variable.